src/TwbsHelper/Form/View/Helper/FormElement.php

```diff
@@ -168,6 +168,31 @@ public function render(ElementInterface $oElement)
         return $sMarkup;
     }
 
+    /**
+     * Render element by helper name
+     *
+     * @param string $name
+     * @param ElementInterface $element
+     * @return string
+     */
+    protected function renderHelper($name, ElementInterface $element)
+    {
+        $helper = $this->getView()->plugin($name);
+
+        if (count($this->options->getValidTagAttributes())) {
```

Comment:
`count` is useless : `if ($this->options->getValidTagAttributes())` is most effective